goldens/public-api/angular_devkit/build_angular/index.md

```diff
@@ -32,6 +32,9 @@ export interface ApplicationBuilderOptions {
     budgets?: Budget_2[];
     clearScreen?: boolean;
     crossOrigin?: CrossOrigin_2;
+    define?: {
+        [key: string]: string;
+    };
```

Comment:
This is awesome. Thanks a lot. A real usecase is e.g. if you use sentry, you can [omit parts of the library with this](https://docs.sentry.io/platforms/javascript/configuration/tree-shaking/#list-of-flags) like `__SENTRY_DEBUG__: false`